examples/automatic_user_loading.py

```diff
@@ -28,7 +28,8 @@ class User(db.Model):
     full_name = db.Column(db.Text, nullable=False)
 
     # NOTE: In a real application make sure to properly hash and salt passwords
-    def check_password(self, password):
+    @staticmethod
```

Comment:
I actually would prefer this to not be static method. It is in this example, but with the `NOTE` there it probably would not be in a real application. This is just some dummy code to kinda show how things might look.

Reply:
Hi @vimalloc 

Thanks for the review! Should I revert the changes here?